### ISSUE TITLE
Support for netstandard1.3 in NuGet package

### ISSUE THREAD

**Issue by cdie** [Pkcs11Interop/Pkcs11Interop#27]
Hi,

First of all, thanks for this amazing library, it saves me a lot of time, and this code is so far cleaner than other implementation.

However, do you think that you can add support for netstandard 1.3 ? Our team is working in this version for some compatibility reasons, and Pkcs11 is not compatible with it. We don't want to fork the whole projet just for this ...

Thanks !

**Comment by jariq** [comment]
All required code should already be in place you just need to use our [DotNetCore.1.0.0.RC2 solution](https://github.com/Pkcs11Interop/Pkcs11Interop/tree/master/src/Pkcs11Interop.DotNetCore). I believe it can be easily adjusted for netstandard1.3 by adding following definition into `frameworks` section in `Pkcs11Interop.DotNetCore\project.json` file:

```json
"netstandard1.3": {
  "buildOptions": {
	"define": [ "COREFX" ]
  },
  "dependencies": {
	"NETStandard.Library": "1.5.0-rc2-24027"
  }
}
```

Please let me know if this works for you.

**Comment by cdie** [comment]
Yes, this works great, but it would be much convenient if the nuget package is compatible with netstandard1.3 (event netstandard1.0). In the mean time, we've generated the solution in 1.3 and created or own nuget package in our company nuget repository as a workaround.

**Comment by jariq** [comment]
I'll consider including assemblies for additional platforms in the next release. So far only net20, net40 and sl5 have been requested. Could you please share more details about the apps/solutions where you are using netstandard1.3 builds? Personally I'm currently using only net40 assemblies so I am curious.

**Comment by cdie** [comment]
We'll use PKCS to read data from PC/SC readers to extract data for a mobile (Xamarin) application, in order to communicate with the french healthcare system. We have a desktop version (which actually works in net452 and great, thanks to your assembly) and we're currently developping a mobile  version. 
We'll have the informations whether from simple reader, which we need PKCS#11, or from "evoluted" readers, than can read the card and send us some JSON.

**Comment by jariq** [comment]
For testing purposes I've created nuget package with the following platforms (subdirectories in lib directory): `net20`, `net40`, `net45`, `nestandard1.3` and `sl5`.

Judging by what you have said in this thread and by what is written in [.NET platform standard description](https://github.com/dotnet/corefx/blob/master/Documentation/architecture/net-platform-standard.md#mapping-the-net-platform-standard-to-platforms) I was expecting that Xamarin projects would reference `netstandard1.3` binaries from this package.

However Xamarin.Android project failed to install this package because of missing `monoandroid` platform and Xamarin.iOS project failed to install this package because of missing `xamarinios` platform. The only type of project that was able to reference `netstandard1.3` binaries was .NET Core project created in Visual Studio 2015.

Now I guess I might be missing something...

Could you please describe in more details how exactly are you using your own netstandard1.3 based nuget package from your company nuget repository?

**Comment by jariq** [comment]
I have just published Pkcs11Interop 3.2.0 along with a [NuGet package](https://www.nuget.org/packages/Pkcs11Interop/3.2.0) that contains binaries for `.NETFramework 2.0`, `.NETFramework 4.0`, `.NETFramework 4.5`, `Silverlight 5.0`, `.NETStandard 1.3`, `MonoAndroid 2.3` and `Xamarin.iOS 1.0` platforms.
